### ISSUE TITLE
Does this repository use the parameter "subdivisions" ?

### ISSUE THREAD

**Issue by watermelon23333** [thtrieu/darkflow#766]
I search "subdivisions" in this repository found that it only show in .cfg file, does this mean that this repository does not use the parameter "subdivisions"? Is this the reason why many people encounter with the issue "OOM" when they use a litter biger batch_size?

**Comment by rauar** [comment]
I don't think subdivion is used. As you said there's no reference to the subdivision parameter in the code. And the other reason I guess is that Tensorflow has no concept of subdivisions. At least if I look at the tf.train.batch there's no such parameter. However I don't know the Tensorflow API that well. And yes: that would explain why increasing the batch size then causes issues (if subdivisions=N would be used increasing the batch size would still make Tensorflow use N inputs in one shot).

I would love to have subdivisions too.

W.r.t OOM: I came to the conclusion that the issue is more the model itself than the input. I read that other people easily use batch sizes of 32 or 64 with 4GB GPUs for yolo-voc in darknet. In darkflow the same model allows me only to use a batch size of 5 at maximum before dying with OOM. And I only have 4 classes and use greyscale images as input so it should use less then normal. So I assume that Tensorflow simply requires more memory compared to darknet's implementation in C focused on one specific use-case.

**Comment by watermelon23333** [comment]
Yes,that is what i want to complain about. My GPU is 11GB 1080ti, but the only way if i want to use yolo in darknet is that set the batchsize to 8!!!
